baybe/utils/validation.py

```diff
@@ -66,3 +75,51 @@ def validator(self: Any, attribute: Attribute, value: Any) -> None:
 
 non_inf_float = _make_restricted_float_validator(allow_nan=True, allow_inf=False)
 """Validator for non-infinite floats."""
+
+
+def get_transform_objects(
```

Comment:
What is the reason why this is not a part of the `dataframe` utility subpackage? Would feel more natural for me to have this in that place.

Reply:
answering in the other thread, closing this one